tests/ubench/CMakeLists.txt

```diff
@@ -4,6 +4,7 @@ include(ExternalProject)
 
 set(bench_sources
     accumulate_functor_values.cpp
+    default_construct.cpp
```

Comment:
Definitely will update `README.md` — I clean forgot to do so. The benchmark is there to demonstrate the lack of necessity (or otherwise) and compiler performance fragility when it comes to trying to finesse this operation, and it is relevant because we are throwing away one of the potential performance benefits of the `memory::vector` class by using the value-constructing allocator.